roles/edpm_bootstrap/tasks/swap.yml

```diff
@@ -0,0 +1,59 @@
+---
+# Copyright 2023 Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+- name: Configure swap file
+  become: true
+  when:
+    - not edpm_bootstrap_swap_partition_enabled|bool
+    - edpm_bootstrap_swap_size_megabytes|int > 0
+  shell: |
```

Comment:
We should probably use `ansible.builtin.shell` for these. I think we were trying to get consistency around the FQCN's with this project.

Reply:
ack, let's keep this for follow-ups